node_modules/oae-activity/tests/test-activity.js

```diff
@@ -1606,7 +1606,7 @@ describe('Activity', function() {
                             assert.ok(_.isObject(activity.actor['tenant']));
 
                             // Ensure only these properties are present
-                            allowedActorProperties = ['oae:id', 'id', 'displayName', 'visibility', 'locale', 'publicAlias', 'picture', 'profilePath', 'resourceType', 'acceptedTC', 'objectType', 'tenant', 'email', 'emailPreference'];
+                            allowedActorProperties = ['oae:id', 'id', 'displayName', 'visibility', 'locale', 'publicAlias', 'picture', 'profilePath', 'resourceType', 'acceptedTC', 'objectType', 'tenant', 'email', 'emailPreference', 'emailChange', 'emailVerified'];
```

Comment:
If we do end up adding more properties to the actor profile, these seem very "meta" and not really needed in the activity model.